src/Compiler/Microsoft.Net.Compilers.Razor.Toolset/build/Microsoft.Net.Compilers.Razor.Toolset.targets

```diff
@@ -0,0 +1,20 @@
+<Project>
```

Comment:
Do you need a buildMultiTargeting folder as well? Our similar toolset packages have them. 

https://learn.microsoft.com/en-us/nuget/create-packages/creating-a-package#include-msbuild-props-and-targets-in-a-package

Reply:
I chatted with @baronfel about this for a bit. For the most part `buildMultiTarget` shouldn't be an issue for us. The toolset here only impacts compilation and that occurs on inner builds. There is no compilation action that runs on outer builds hence this is not functionally needed for core scenarios. 

On the other hand there are tools that run on outer builds that reasonably may want to look at compiler arguments. For example our [package validation tool](https://github.com/dotnet/sdk/pull/19669) does exactly this. 

Essentially it's _more correct_ to include a `buildMultiTarget` here and I think we should do so to avoid future hiccups.

Reply:
I'm not entirely sure what you're asking for: should I be putting my target in the `buildMultiTarget` folder _instead_ of `build`, or in addition to?

Reply:
I can't think of a scenario where that would matter, but it also shouldn't hurt to do it.